pkg/cluster/cluster.go

```diff
@@ -491,6 +495,53 @@ func compareResourcesAssumeFirstNotNil(a *v1.ResourceRequirements, b *v1.Resourc
 
 }
 
+func (c *Cluster) validateResources(spec *acidv1.PostgresSpec) error {
+
+	const (
+		cpuMinimum     = "256m"
```

Comment:
The enforcement of such limits and  their values should imo be configurable.

Reply:
I think in this case it's better if we define what's good for the operator. Note, that only the `spec` is validated. You could still configure lower default values and they will be used if you do not specify requests/limits in the cluster manifest.